tests/src/Common/external/external.depproj

```diff
@@ -84,14 +82,14 @@
     <ProjectAssetsFile>$(SourceDir)Common\external\obj\project.assets.json</ProjectAssetsFile>
   </PropertyGroup>
 
-  <Target Name="AddXunitConsoleRunner" BeforeTargets="ResolveReferences" Condition="'$(TargetGroup)' == 'netfx'">
-    <Error Condition="!Exists('$(PackagesDir)$(XUnitRunnerPackageId)\$(XUnitPackageVersion)\tools\xunit.console.exe')"
-            Text="Error: looks the package $(PackagesDir)$(XUnitRunnerPackageId)\$(XUnitPackageVersion) not restored or missing xunit.console.exe."
+  <Target Name="AddXunitConsoleRunner" BeforeTargets="ResolveReferences">
+    <Error Condition="!Exists('$(PackagesDir)\$(XUnitRunnerPackageId)\$(XUnitPackageVersion)\tools\netcoreapp1.0\xunit.console.dll')"
```

Comment:
Should `netcoreapp1.0` be defined in a props file or this is the version we use now and forever?

And below.

Reply:
I updated this to use a property in the same file for now, since this is specific to the xunit console runner package layout. The package ships with assets for netcoreapp1.0 and netcoreapp2.0. I'm not sure where else the runner might be used and I see a bunch of projects in our tree that haven't been updated to netcoreapp2.0, so I'm using this as a lowest common denominator.